versioner/nonsemanticversion.go

```diff
@@ -0,0 +1,108 @@
+package versioner
```

Comment:
who imports package versioner? i'm unsure how this fits into the build system

also, i think one-off go packages probably deserve either their own repos, or to be put in compute-image-tools. this repo isn't set up for go development

Reply:
yeah, i totally agree that this does not belong here. I wanted this to be in a new repo but I wanted to check with you before moving ahead with requesting a new repo with the security team.
Let's meet tomorrow. I can explain you my end to end idea with the auto-versioner and also i want some clarification on requirements.